src/druid628/exactTarget/EtClient.php

```diff
@@ -28,7 +28,7 @@ class EtClient extends EtBaseClass
     const SOAPWSDL = "http://exacttarget.com/wsdl/partnerAPI";
 
     /** @var \druid628\exactTarget\EtSoapClient $client */
-    protected $client;
+    public $client;
```

Comment:
Gotcha. If you'll change that back to protected I'll go ahead and merge this.
Thanks so much for this fix.